src/sentry/static/sentry/app/views/groupDetails/shared/groupTagValues.jsx

```diff
@@ -47,46 +47,42 @@ const GroupTagValues = createReactClass({
     }
   },
 
-  fetchData() {
+  async fetchData() {
     const {params, query} = this.props;
 
     this.setState({
       loading: true,
       error: false,
     });
 
-    this.api.request(`/issues/${params.groupId}/tags/${params.tagKey}/`, {
-      query,
-      success: data => {
-        this.setState({
-          tagKey: data,
-          loading: this.state.tagValueList === null,
-        });
-      },
-      error: error => {
-        this.setState({
-          error: true,
-          loading: false,
-        });
-      },
-    });
-
-    this.api.request(`/issues/${params.groupId}/tags/${params.tagKey}/values/`, {
-      query,
-      success: (data, _, jqXHR) => {
-        this.setState({
-          tagValueList: data,
-          loading: this.state.tagKey === null,
-          pageLinks: jqXHR.getResponseHeader('Link'),
-        });
-      },
-      error: error => {
-        this.setState({
-          error: true,
-          loading: false,
-        });
-      },
-    });
+    const promises = [
+      this.api.requestPromise(`/issues/${params.groupId}/tags/${params.tagKey}/`, {
+        query,
+      }),
+      this.api.requestPromise(`/issues/${params.groupId}/tags/${params.tagKey}/values/`, {
+        query,
+        includeAllArgs: true,
+      }),
+    ];
+
+    try {
+      const [tagKey, tagValueResponse] = await Promise.all(promises);
+      // eslint-disable-next-line no-unused-vars
+      const [tagValueList, _, jqXhr] = tagValueResponse;
```

Comment:
You don't use jqXhr why do you unpack it?

Reply:
Oops, thanks, missing pagination